packages/evm/src/fungible.ts

```diff
@@ -197,6 +201,8 @@ class EvmFungibleAssetTransfer extends BaseTransfer {
     const bridge = Bridge__factory.connect(domainConfig.bridge, provider);
     const fee = await this.getFee();
 
+    await this.verifyAccountBalance(fee);
```

Comment:
I think its better to make this call in `isValidTransfer` as `getApprovalTransactions` also requires the account to be funded

Reply:
@saadjhk I'm thinking about where to put the validation and why I call the method later as I want to ensure the balance is correct before the on-chain interaction. I imagine that some User who will integrate our SDK and has a UI could create an instance of Fungible transfer, but make a call later. So the balance validation could be changed. Also to keep the principle of single responsibility for the method I would call it outside the "isValidTransfer" as this method makes sure that the provided combination of the source and destination data are valid with our config.